CHANGELOG.md

```diff
@@ -4,6 +4,90 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2024-07-23
+
+### Migrating from 0.4.0 -> 0.6.0
+
+Module Imports
+
+```
+use deepgram::{
+---    transcription::prerecorded::{
++++    common::{
+        audio_source::AudioSource,
+        options::{Language, Options},
+    },
+    Deepgram, DeepgramError,
+};
+```
+
```

Comment:
Also mention here that some option enums have changed, and will need to be updated.